.github/workflows/build_wheels_linux.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       matrix:
         include:
           - repository: pytorch/data
-            pre-script: ""
+            pre-script: packaging/pre_build_script_linux.sh
             post-script: ""
```

Comment:
Is there a way to validate the binary is manylinux compatible? In post-script?

Reply:
Can we use [this logic](https://github.com/pytorch/data/blob/0a0ae5d35be439c786c9ab798ea1596238d8640a/.github/workflows/_build_test_upload.yml#L162-L186) to validate manylinux?

Reply:
Yeah something like that
```py
pip3 install auditwheel
auditwheel show <path_to_wheel>
```

Reply:
I'll add it to the post-script.